core/templates/pages/exploration-player-page/services/hints-and-solution-manager.service.ts

```diff
@@ -179,9 +179,9 @@ angular.module('oppia').factory('HintsAndSolutionManagerService', [
         }
 
         wrongAnswersSinceLastHintConsumed++;
-        if (!areAllHintsExhausted() && !isAHintWaitingToBeViewed()) {
+        if (!areAllHintsExhausted()) {
           if (numHintsReleased === 0 &&
```

Comment:
Break after the (

This is the standard practice we follow.

Reply:
Done.